README.md

```diff
@@ -40,43 +40,6 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
 
 **Inductiva API** contains pre-built scenarios that define physical systems of interest ready to simulate. Users can choose some parameters and configure the system according to their needs, run the simulation using the most adequate resources and visualize the results.
 
-Let's go through the currently available scenarios in **Inductiva API**.
-
-### Protein solvation
-
-The protein solvation scenario models the dynamics of a protein whose structure is described by a PDB file. The protein is placed in a cubic box filled with water. If the protein has a non-zero electric charge, charged ions are added to the solution to neutralize the system. First, the system undergoes an [energy minimization](https://manual.gromacs.org/current/reference-manual/algorithms/energy-minimization.html) process to eliminate any steric clashes or structural issues within the protein-water system. After this, the position of the atoms in this system is updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields.
-
-#### Example
-
-Initialize the scenario:
-
-```python
-from inductiva import molecules
```

Comment:
Keep the example code in the main page (even if it is duplicated elsewhere) as a way of illustrating how scenarios work. It follows naturally from the previous paragraph.